v1/examples/life-events/death-registration-update.json

```diff
@@ -0,0 +1,83 @@
+{
+  "iss": "https://scim.example.com",
+  "iat": 1691637001,
+  "jti": "4d3559ec67504aaba65d40b0363faad8",
+  "toe": 618772150,
+  "events": {
+    "https://ssf.account.gov.uk/v1/deathRegistrationUpdate": {
+      "subject": {
+        "name": [
+          {
+            "nameParts": [
+              {
+                "value": "Velma",
+                "type": "GivenName"
+              },
+              {
+                "value": "Rogers",
+                "type": "FamilyName"
+              }
+            ]
+          },
+          {
+            "nameParts": [
+              {
+                "value": "Velma",
+                "type": "GivenName"
+              },
+              {
+                "value": "Smith",
+                "type": "FamilyName"
+              }
+            ],
+            "description": "Otherwise or formerly known as"
+          },
+          {
+            "nameParts": [
+              {
+                "value": "Velma",
+                "type": "GivenName"
+              },
+              {
+                "value": "Molone",
+                "type": "FamilyName"
+              }
+            ],
+            "description": "Name before marriage"
```

Comment:
Are we proposing mapping the GRO maiden name field to this? Do we need a way of ascribing machine readable meaning to this description, or will it only be used for manual interventions?

Reply:
So I think the _current_ proposal is that we would map to this description but discourage (implicitly?) this from being used except in a manual matching process. But if DWP tell us that they need something machine-readable because they want to map to a specific field then we would probably come up with something else (ISTM this would be an optimisation, but eh).